velox/docs/configs.rst

```diff
@@ -547,6 +547,14 @@ Each query can override the config by setting corresponding query session proper
      - string
      -
      - The GCS service account configuration as json string.
+   * - hive.gcs.max-retry-count
+     - integer
+     -
+     - The GCS maximum retry counter of transient errors.
+   * - hive.gcs.max-retry-time
```

Comment:
We recently added a timeout for s3 using string. https://github.com/facebookincubator/velox/pull/9472
We could do the same here. Turns out it is straightforward.

Reply:
Done!